airlock/emails.py

```diff
@@ -5,16 +5,24 @@
 from jobserver.models import Release
 
 
+def get_email_context(airlock_event):
+    # The first update is the event itself, don't include this in emails
+    updates = airlock_event.describe_updates()[1:]
+    return {
+        "release_request_id": airlock_event.release_request_id,
+        "request_author": airlock_event.request_author.name,
+        "workspace": airlock_event.workspace.name,
+        "updates": updates,
+    }
+
+
 def send_request_released_email(airlock_event):
     release = Release.objects.get(id=airlock_event.release_request_id)
     f = furl(settings.BASE_URL)
     f.path = release.get_absolute_url()
 
-    context = {
-        "url": f.url,
-        "request_author": airlock_event.request_author.name,
-        "workspace": airlock_event.workspace.name,
-    }
+    context = get_email_context(airlock_event)
+    context["url"] = (f.url,)
```

Comment:
Is this meant to be a tuple? Might be a stray trailing comma left over after the refactor.

Reply:
It is, thanks